src/srb/decompressor.cpp

```diff
@@ -71,15 +71,20 @@ namespace srb
             { { 80, 75, 3, 4 }, ZIP },
             { { 120, -100 }, ZLIB }
         };
-
+        
+	/*
+	* If the first line of the VCF file has less than 5 characters, 
```

Comment:
VCF xD,  this tool is not only for vcf files. that was vcf-validator which was for vcf files. it is made generic for all kind of files.

Reply:
Ah I see xD